### PR TITLE
Added delete for admins only

### PR DESCRIPTION
**Pull request by jbolles94**

### CODE REVIEW

File: src/cloudStorage/controller.js [hunk 1]
```diff
@@ -19,21 +23,39 @@ async function generateSignedUploadURL(req, res) {
...
-      const [url] = await storage
-        .bucket(CLOUD_STORAGE_BUCKET_NAME)
-        .file(location)
-        .getSignedUrl(options);
-
-      return { data: { url } };
+      return { data: { url: await generateSignedURL("write", location) } };
     },
     res,
     "fetching packing slips"
```
Comment (joshplumley) [inline src/cloudStorage/controller.js:29]: change

File: src/cloudStorage/controller.js [hunk 2]
```diff
@@ -19,21 +23,39 @@ async function generateSignedUploadURL(req, res) {
...
   );
 }
+
+async function deleteCloudStorageObject(filepath) {
+  const deleteOptions = {
+    ifGenerationMatch: generationMatchPrecondition,
```
Comment (joshplumley) [inline src/cloudStorage/controller.js:35]: `generationMatchPrecondition` doesnt seem to be defined anywhere

File: src/cloudStorage/controller.js (hunk 1 above)
Comment (jbolles94) [reply src/cloudStorage/controller.js]: Changed.

File: src/cloudStorage/controller.js (hunk 2 above)
Comment (jbolles94) [reply src/cloudStorage/controller.js]: This is no longer here

**Review by joshplumley — APPROVED** [review]

**Pull request merged**